### PR TITLE
Rewrite deployment article

### PR DESCRIPTION
**Pull request by daniel-schroeder-zeiss**

### CODE REVIEW

File: docs/deployment.md [hunk 1]
```diff
@@ -17,63 +17,59 @@ Inhalt:
...
+><GeneratePluginPackageOnBuild>true</GeneratePluginPackageOnBuild>
+>```
+
+## Distributing a plug-in package
+Unfortunately, there is currently no package management infrastructure available that could be used to manage, share and distribute packages centrally. You can, however, manage and distribute packages by your own means.
+
+## Installing a plug-in package
+Plug-in packages can directly be installed in *PiWeb Auto Importer* in two different ways:
+1. by dropping the package file to install anywhere in *PiWeb Auto Importer*, or
+1. by opening the plug-in management view via `File > Plug-ins...` and then clicking on `Install plug-in...` in the management actions dropdown. You will then be prompted to choose the package file to install.
```
Comment (stephan-scholz-zeiss) [inline docs/deployment.md:43]: Here “1.” is used for the second time.

File: docs/deployment.md [hunk 2]
```diff
@@ -17,63 +17,59 @@ Inhalt:
...
+
+*PiWeb Auto Importer* will show an overview of the plug-in to install. You can continue by pressing the `Install` button.
+
+![Install window](../assets/images/deployment/ai_install_2.png "Install window")
+
+Now the new plug-in is listed in the plug-in management view. However, it is not active yet since a restart is required to actually install and load it. At this point you can use the opportunity to install or remove further plug-ins before finally pressing the `Restart now` button in the info belt at the bottom of the window.
+
+![Restart request](../assets/images/deployment/ai_install_3.png "Restart request")
+
+Immediately after the restarting, the plug-in installer opens. It displays all pending installation operations. You can press the `Run now` button to apply all these changes. At this point the installer prompts for administrative privileges if you do not already have them, stops all import plans running as windows services and applies the plug-in changes. Afterwards, all previously stopped windows services will be started again. When everything is done, you can close the window and *PiWeb Auto Importer* starts normally.
```
Comment (stephan-scholz-zeiss) [inline docs/deployment.md]: "the restarting" sounds strange. Maybe:
The plug-in installer opens immediately after the restart.

File: docs/deployment.md [hunk 3]
```diff
@@ -17,63 +17,59 @@ Inhalt:
...
+
+![Plug-in Installer](../assets/images/deployment/ai_install_4.png "Plug-in Installer")
+
+{: .note}
+Pressing the `Discard` button closes the installer, drops all scheduled plug-in operations and starts *PiWeb Auto Importer* without changing the plug-in setup. Pressing the `Later` button instead will also close the installer and start *PiWeb Auto Importer*. However, the scheduled plug-in operations are not discarded. You will be prompted again to apply them after the next restart.
+
+<!-- ### Download compiled example plug-in
```
Comment (stephan-scholz-zeiss) [inline docs/deployment.md]: The comment can certainly be removed.

File: docs/deployment.md (hunk 1 above)
Comment (daniel-schroeder-zeiss) [reply docs/deployment.md:43]: That is actually intended. Jekyll takes care of the correct numbering when rendering. This makes rearranging easier.
grafik

File: docs/deployment.md (hunk 3 above)
Comment (daniel-schroeder-zeiss) [reply docs/deployment.md]: Yes.

File: docs/deployment.md (hunk 2 above)
Comment (daniel-schroeder-zeiss) [reply docs/deployment.md]: Rewritten.

**Review by stephan-scholz-zeiss — APPROVED** [review]

**Pull request merged**